### PR TITLE
implement KillTimer

### PR DESCRIPTION
**Pull request by LinusU**

### CODE REVIEW

File: win32/src/winapi/user32/timer.rs [hunk 1]
```diff
@@ -54,6 +54,17 @@ impl Timers {
     }
 }
 
+#[win32_derive::dllexport]
+pub fn KillTimer(machine: &mut Machine, hWnd: HWND, uIDEvent: u32) -> bool {
+    let timers = &mut machine.state.user32.timers.0;
+    if let Some(index) = timers.iter().position(|t| t.hwnd == hWnd && t.id == uIDEvent) {
+        timers.remove(index);
```
Comment (evmar) [inline win32/src/winapi/user32/timer.rs:61]: I think this can be timers.swap_remove

**Comment by LinusU** [comment]
@evmar rebased on main, and adressed comment ✅

**Comment by evmar** [comment]
In retrospect sorry for nitpicking swap_remove, I am sure the perf doesn't matter here and if it did we'd probably use a different data structure!  Thanks for fixing it anyway.

**Comment by LinusU** [comment]
No worries!

**Pull request merged**